shared/httpHandler/queryBuilder.js

```diff
@@ -0,0 +1,25 @@
+function queryBuilder({ query }) {
+	const parts = [];
+
+	for (const key of Object.keys(query)) {
+		const value = query[key];
+		const escapedKey = encodeURIComponent(key);
+
+		if (Array.isArray(value)) {
+			value.forEach(arrayValue => {
+				parts.push(`${escapedKey}=${encodeURIComponent(arrayValue)}`);
+			});
+		} else {
+			let qsEntry = escapedKey;
+
+			if (value || typeof value === 'string') {
+				qsEntry += `=${encodeURIComponent(value)}`;
+			}
+			parts.push(qsEntry);
+		}
+	}
+
+	return parts.join('&');
+}
```

Comment:
@chulanovskyi-bs I'm a bit under the impression we kind of do query encoding without using the standard apis that would help there see https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams (we already use it in the browser side if i recall well)

I guess this is  working but maybe we should envision leveraging web apis when they are available :)

Reply:
Huh, indeed, this module can be completely removed, thank you

Reply:
[Added here](https://github.com/hackolade/glue/pull/49/commits/ea27b561516382d68fb0e9ce48e271fac9ab4f16)